.github/workflows/generate.yml

```diff
@@ -57,15 +57,14 @@ jobs:
       - name: Re-generate isp
         run: ./run.sh
 
-      - name: Re-generate isp-slates
+      - name: Re-generate isp-slate
```

Comment:
nit: it's called the `Slates API` I wonder if it makes more sense to change the folder to `isp-slates` rather than change the step to `isp-slate`?

Reply:
Will do in a separate PR shortly!